### DISCUSSION TITLE
Translate contact form into excelsheet

### DISCUSSION THREAD

**Discussion by nathalieverhaert** [craftcms/contact-form#234] [Ideas]
Hi guys, is it possible to send the information submitted in the contact form into an excel sheet instead of sending an e-mail.

**Comment by AugustMiller** [comment]
You can send contact form submissions to any other service using an even handler. The logic for working with Excel is specific to whatever APIs it provides.

- [Events docs](https://github.com/craftcms/contact-form#the-aftersend-event)
- [Description of generating a CSV](https://github.com/craftcms/contact-form/issues/153) of a submission
